rosco-web/config/packer/install_packages.sh

```diff
@@ -1,25 +1,47 @@
 #!/bin/bash
 
-echo "deb_repo=$deb_repo"
+# Strip the first part to avoid credentials leaks
```

Comment:
This file previously used 2-space indentation. It now appears to have a combination of 3-space and 4-space indentation. Would prefer to keep it at 2-spaces everywhere in this file.